packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -365,11 +358,6 @@ export async function zipCode({ dependenciesFolder, humanInTheLoopFlag, modulePa
             fs.rmSync(dependenciesFolder.path, { recursive: true, force: true })
         }
     } catch (e: any) {
-        telemetry.codeTransform_logGeneralError.emit({
```

Comment:
This isn't deprecated right?

Reply:
right, the `logGeneralError` metric isn't deprecated, but I think emitting it here isn't very useful since we don't look at this metric, and user journey will track the success rate of upload step which includes zipping. I can add it back if you think the metric will help with troubleshooting, though customers should be able to see the error in log when requesting help.